src/Avalonia.Visuals/Media/Color.cs

```diff
@@ -41,6 +46,18 @@ static Color()
         /// </summary>
         public byte B { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="Color"/> is uninitialized and
+        /// considered null.
+        /// </summary>
+        public bool IsEmpty
```

Comment:
Again, from WPF and needed in certain advanced scenarios to differentiate an uninitialized color from Transparent (note that transparent is #00FFFFFF instead of #00000000.

https://docs.microsoft.com/en-us/dotnet/api/system.drawing.color.isempty?view=net-6.0

Reply:
Marking resolved since no additional comments in review